RecoTracker/TrackProducer/interface/TrackProducerBase.icc

```diff
@@ -162,7 +162,7 @@ TrackProducerBase<T>::setSecondHitPattern(Trajectory* traj, T& track,
   std::unique_ptr<Propagator> localProp(prop->clone());
 
   //use negative sigma=-3.0 in order to use a more conservative definition of isInside() for Bounds classes.
-  Chi2MeasurementEstimator estimator(30.,-3.0,0.5,2.0,0.5);  // same as defauts....
+  Chi2MeasurementEstimator estimator(30.,-3.0,0.5,2.0,0.5,std::numeric_limits<float>::max());  // same as defauts....

```

Comment:
This piece is different wrt. the 81X PR, where the last parameter is `0.9`, in order to minimize the changes to default RECO.